examples/input2.py

```diff
@@ -0,0 +1,65 @@
+from helpers import render_example
+
+"""
+This example demonstrates slightly more complicated mouse and keyboard input
+in the interactive runtime viewport.
+"""
+
+from neoscore.common import *
+
+annotation = """
```

Comment:
Is this something you wanted to include in a displayed RichText object? Otherwise it can just be in a docstring/comment with out HTML markup

Reply:
I copied the "annotation" HTML markups from the original input.py file. It makes more sense for this to just be a comment in the new example I think.